samples/snippets/quickstart.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+A sample script showing how to create, list and delete Google Compute Engine
+instances using the google-cloud-compute library. It can be run from command
+line to create, list and delete an instance in a given project in a given zone.
+"""
+
+import argparse
+
+# [START compute_instances_list]
+# [START compute_instances_list_all]
+# [START compute_instances_create]
+# [START compute_instances_delete]
+# [START compute_instances_operation_check]
+import typing
+
+import google.cloud.compute_v1 as gce
```

Comment:
Style suggestion for consistency with existing python samples.
```suggestion
from google.cloud import compute_v1
```